### PR TITLE
Add nose mask publisher

### PR DESCRIPTION
**Pull request by itohdak**
Enable `people_mask_publisher.py` to receive `"Nose"` as `limb_part` parameter.
Also modify `sample_people_pose_estimation_2d.launch` so that the results with the example bagfile can be viewed with following command.
`$ roslaunch jsk_perception sample_people_pose_estimation_2d.launch LIMB_PART:="Nose"`

people_mask_publisher

### CODE REVIEW

**Comment by knorth55** [comment]
Can you put visualization images?
Also can you update documentation with visualization images and how to run sample launch?

**Comment by itohdak** [comment]
I think a documentation for `people_mask_publisher.py` doesn't exist yet.
Should I create a new documentation or add a sample launch into `people_pose_estimation_2d.py` documentation?

Cc: @iory

**Comment by knorth55** [comment]
Yes, please.

**Comment by iory** [comment]
> I think a documentation for people_mask_publisher.py doesn't exist yet.
Should I create a new documentation or add a sample launch into people_pose_estimation_2d.py documentation?

@itohdak 
You can create a new documentation for `people_mask_publisher.py`

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
+people_mask_publisher.py
+============================
+
+
+What is this?
+-------------
+
+.. image:: ./images/people_pose_estimation_2d.png
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: I added a visualization image. Please check it.

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+* ``~input`` (``sensor_msgs/Image``)
+
+  Input image.
+
+* ``~input/pose`` (``jsk_recognition_msgs/PeoplePoseArray``)
+
+  Input people pose array.
+
+* ``~input/info`` (``sensor_msgs/CameraInfo``)
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: Thank you for your review. I removed this line.

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  Used when ``limb_part`` is ``Nose``.
+
+* ``face_width_margin_ratio`` (Float, Default: ``1.3``)
+
+  Ratio that is used to decide face width margin.
+
+  Used when ``limb_part`` is ``Nose``.
+
+* ``~approximate_sync`` (Bool, Default: ``True``)
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: You are right. Fixed.

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+  Slop for approximate sync.
+
+
+Example
+-------
+
+.. code-block:: bash
+
+   roslaunch jsk_perception sample_people_pose_estimation_2d.launch GPU:=0 limb_part:='RHand'
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: Fixed.

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  Debug image which has visualized rectangle of the masked limb part.
+
+Parameters
+----------
+
+* ``~person_indices`` (Int, Default: ``-1``)
+
+  Which person to create mask of. ``-1`` represents creating masks of all people.
+
+* ``~limb_part`` (String, Default: ``all``)
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: It's true. I added ```List of String```.
I also added a description of allowed limb part.

File: doc/jsk_perception/nodes/people_mask_publisher.rst [hunk 6]
```diff
@@ -0,0 +1,103 @@
+people_mask_publisher.py
+============================
```
[the remark this answers is not in the document]
Comment (itohdak) [reply doc/jsk_perception/nodes/people_mask_publisher.rst]: I got it. I fixed it.

**Comment by knorth55** [comment]
@k-okada travis passed!
can you check and merge this PR?

**Pull request merged**